src/map/status.cpp

```diff
@@ -11255,6 +11327,39 @@ int status_change_start(struct block_list* src, struct block_list* bl,enum sc_ty
 			val1 = 15; // Heal Power rate bonus
 			val2 = 30; // SP Recovery rate bonus
 			break;
+		case SC_SUNSTANCE:
+			val2 = 2 + val1;// ATK Increase
+			tick = INFINITE_TICK;
+			break;
+		case SC_LUNARSTANCE:
+			val2 = 2 + val1;// MaxHP Increase
+			tick = INFINITE_TICK;
+			break;
+		case SC_STARSTANCE:
+			val2 = 4 + 2 * val1;// ASPD Increase
+			tick = INFINITE_TICK;
+			break;
+		case SC_UNIVERSESTANCE:
+			val2 = 2 + val1;// All Stats Increase
+			tick = INFINITE_TICK;
+			break;
+		case SC_NEWMOON:
+			val2 = 7;// Number of Regular Attacks Until Reveal
+			break;
+		case SC_FALLINGSTAR:
+			val2 = 8 + 2 * (1 + val1) / 2;// Autocast Chance
+			if (val1 >= 7)
+				val2 += 1;// Make it 15% at level 7.
```

Comment:
Should this be `val2 -= 1`? it's a little weird to me because of maybe there's some operator precedence it's expecting. but what i see it'd do 8 + 2*8/2 => 16 ..